src/shared/seccomp-util.c

```diff
@@ -47,6 +47,8 @@ uint32_t seccomp_local_archs[] = {
                 SCMP_ARCH_AARCH64,     /* native */
 #elif defined(__arm__)
                 SCMP_ARCH_ARM,
+#elif defined(__loongarch_lp64)
```

Comment:
Why not `__loongarch64` ?

Reply:
Please read the end of this document, https://loongson.github.io/LoongArch-Documentation/LoongArch-toolchain-conventions-EN.html#_cc_preprocessor_built_in_macro_definitions

Reply:
`if defined(__loongarch64)` is used elsewhere, do those need to be updated?

Reply:
It is best if __loongarch64 can be updated, and if it cannot, it will still be available at least for a while

Reply:
could you please add a commit to do so then?

Reply:
Done.